examples/Embeddings/Example04_SimpleEmbeddingProtocol.cs

```diff
@@ -0,0 +1,43 @@
+﻿using NUnit.Framework;
+using OpenAI.Embeddings;
+using System;
+using System.ClientModel;
+using System.Text.Json;
+
+namespace OpenAI.Examples;
+
+public partial class EmbeddingExamples
+{
+    [Test]
+    public void Example04_SimpleEmbeddingProtocol()
```

Comment:
Should we add (maybe just as a link) this sample to the README?

Reply:
I do think we should improve discovery of our examples by linking them in the README. I would like to do it as a bigger sweep in a separate PR.